Box.V2.Test.Integration/Configuration/Commands/DisposableCommands/CreateRetentionPolicyCommand.cs

```diff
@@ -27,9 +28,27 @@ public async Task<string> Execute(IBoxClient client)
                 RetentionLength = 1,
                 DispositionAction = DispositionAction.permanently_delete.ToString(),
             };
+            try
+            {
+                var response = await client.RetentionPoliciesManager.CreateRetentionPolicyAsync(retentionPolicyRequest);
+                Policy = response;
+            }
+            catch
+            {
+                // TODO: 12-09-2022, @mcong
+                // There is an error on backend side, which will return 409 status code "conflict"
+                // but retention policy still created.
+                // Delete this try-catch after the issue is fixed.
+                var policies = await client.RetentionPoliciesManager.GetRetentionPoliciesAsync(_policyName);
+                if (policies.Entries.Count == 1)
```

Comment:
That workaround will do for now. We should probably also do something when Count is not equal to 1 for some reason. Maybe throw an exception?

Reply:
Fixed. I throw the same exception if the policy count is other than 1.